scripts/e2e_check_pod_restarts.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+restarts=$(kubectl get pods -n mayastor | grep -e mayastor -e moac | awk '{print $4}')
```

Comment:
An example of what the grep matches would be useful.

Reply:
Done